insights/specs/default.py

```diff
@@ -625,6 +625,7 @@ class DefaultSpecs(Specs):
     sysctl_conf = simple_file("/etc/sysctl.conf")
     sysctl_d_conf_etc = glob_file("/etc/sysctl.d/*.conf")
     sysctl_d_conf_usr = glob_file("/usr/lib/sysctl.d/*.conf")
+    sys_fs_cgroup_memory_tasks_number = simple_command("/usr/bin/find /sys/fs/cgroup/memory -name 'tasks' | wc -l")
```

Comment:
The pipeline "|" is not supported by insights collection, you may need to convert this spec to a `datasource` spec.

Reply:
@xiangce - Updated to use datasource, please have a check.